core/server/api/app.js

```diff
@@ -11,7 +11,7 @@ var debug = require('debug')('ghost:api'),
     // API specific
     auth = require('../auth'),
     cors = require('../middleware/api/cors'),   // routes only?!
-    spamPrevention = require('../middleware/api/spam-prevention'), // routes only
```

Comment:
Ah I can remove both instances of spam prevention, the private-blogging method is not used now, in favour of brute-express so I will update for this.

Also, I changed my mind regarding the export methods but forgot to revert the code so I will tweak that a bit too :)